hi-ml/src/health_ml/utils/box_utils.py

```diff
@@ -0,0 +1,132 @@
+from dataclasses import dataclass
+from typing import Optional, Sequence, Tuple
+
+import numpy as np
+
+@dataclass(frozen=True)
+class Box:
```

Comment:
no test coverage for this class?

Reply:
@dccastro is writing tests for this right now, they will be merged to RadiomicsNN sometimes soon